### PR TITLE
8277159: Fix java/nio/file/FileStore/Basic.java test by ignoring /run/user/* mount points

### PR DESCRIPTION
**Pull request by akashche**
…/user/* mount points

The fix to the test is the same as in #5136 . The possible problem with this test was identified in [this comment](https://github.com/openjdk/jdk/pull/5136#issuecomment-900402107), however FileSystemException is thrown instead of NoSuchFileException in this case, thus the special check for "/run/user" is added.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8277159](https://bugs.openjdk.java.net/browse/JDK-8277159): Fix java/nio/file/FileStore/Basic.java test by ignoring /run/user/* mount points

### Reviewers
 * [Brian Burkhalter](https://openjdk.java.net/census#bpb) (@bplb - **Reviewer**)
 * [Aleksey Shipilev](https://openjdk.java.net/census#shade) (@shipilev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6399/head:pull/6399` \
`$ git checkout pull/6399`

Update a local copy of the PR: \
`$ git checkout pull/6399` \
`$ git pull https://git.openjdk.java.net/jdk pull/6399/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6399`

View PR using the GUI difftool: \
`$ git pr show -t 6399`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6399.diff">https://git.openjdk.java.net/jdk/pull/6399.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/nio/file/FileStore/Basic.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
  */
 
 /* @test
- * @bug 4313887 6873621 6979526 7006126 7020517 8264400
+ * @bug 4313887 6873621 6979526 7006126 7020517 8264400 8277159
```
Comment (bplb) [inline test/jdk/java/nio/file/FileStore/Basic.java:25]: I don't think that the bug ID needs to be added here as this is a change to the test itself, only.

Comment (akashche) [reply test/jdk/java/nio/file/FileStore/Basic.java]: Thanks for your comments! I've removed the bug ID.

**Review by bplb — APPROVED** [review]
Passed 5 test repeats on Linux, macOS, and Windows. Approved.

**Comment by akashche** [comment]
Thanks for the review!

**Review by shipilev — APPROVED** [review]

**Pull request closed without merging**